City-of-Helsinki/berth-reservations-admin
# VEN-1001 | Add contract info to application details components

### Closes :no_good_woman:

[VEN-1001](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1001)

## Testing :alembic:

### Manual testing :construction_worker_man:

- Handling either a berth or WS contract should add the generated contract details to the application details
- Status should update accordingly, beginning with "Allekirjoitus puuttuu"
- Should be able to view the document through the link
